syz-ci/jobs.go

```diff
@@ -366,6 +366,7 @@ func (jp *JobProcessor) process(job *Job) *dashapi.JobDoneReq {
 	*mgrcfg = *mgr.managercfg
 	mgrcfg.Workdir = filepath.Join(dir, "workdir")
 	mgrcfg.KernelSrc = filepath.Join(dir, "kernel")
+	mgrcfg.KernelBuildSrc = mgr.kernelBuildSrc
```

Comment:
Builds for jobs are done in a different location from kernel builds for normal fuzzing.